monkey/infection_monkey/pyinstaller_hooks/hook-infection_monkey.ransomware.py

```diff
@@ -0,0 +1,4 @@
+from PyInstaller.utils.hooks import collect_data_files, collect_submodules
+
+hiddenimports = collect_submodules("infection_monkey.ransomware")
+datas = collect_data_files("infection_monkey.ransomware", include_py_files=True)
```

Comment:
Why do we need `include_py_files=True`? Python files are included automatically

Reply:
I copy-pasted from `hook-infection_monkey.exploit.py`. I'll investigate whether or not this is needed.